notebooks/generate_synthetic_data.ipynb

```diff
@@ -33,165 +32,11 @@
-      "\u001b[?25hInstalling collected packages: patool\n",
-      "Successfully installed patool-1.12\n"
-     ]
-    }
-   ],
-   "source": [
-    "!pip install patool"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 21,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "import patoolib"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 25,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "patool: Extracting ./fonts/pizza-is-my-favorite.zip ...\n",
-      "patool: ... ./fonts/pizza-is-my-favorite.zip extracted to `./fonts/'.\n"
-     ]
-    },
-    {
-     "data": {
-      "text/plain": [
-       "'./fonts/'"
-      ]
-     },
-     "execution_count": 25,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "patoolib.extract_archive('./fonts/pizza-is-my-favorite.zip',outdir='./fonts/')"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 29,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "patool: Extracting ./fonts/acki-preschool.zip ...\n",
-      "patool: ... ./fonts/acki-preschool.zip extracted to `./fonts/'.\n",
-      "patool: Extracting ./fonts/bpchildlefty.zip ...\n",
-      "patool: ... ./fonts/bpchildlefty.zip extracted to `./fonts/'.\n",
-      "patool: Extracting ./fonts/schoolbell.zip ...\n",
-      "patool: ... ./fonts/schoolbell.zip extracted to `./fonts/'.\n",
-      "patool: Extracting ./fonts/tommys-first-alphabet.zip ...\n",
-      "patool: ... ./fonts/tommys-first-alphabet.zip extracted to `./fonts/'.\n",
-      "patool: Extracting ./fonts/acki-preschool.zip ...\n",
-      "patool: ... ./fonts/acki-preschool.zip extracted to `./fonts/'.\n",
-      "patool: Extracting ./fonts/two-turtle-doves.zip ...\n",
-      "patool: ... ./fonts/two-turtle-doves.zip extracted to `./fonts/'.\n"
-     ]
-    },
-    {
-     "data": {
-      "text/plain": [
-       "'./fonts/'"
-      ]
-     },
-     "execution_count": 29,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "patoolib.extract_archive('./fonts/acki-preschool.zip',outdir='./fonts/')\n",
-    "patoolib.extract_archive('./fonts/bpchildlefty.zip',outdir='./fonts/')\n",
-    "patoolib.extract_archive('./fonts/schoolbell.zip',outdir='./fonts/')\n",
-    "patoolib.extract_archive('./fonts/tommys-first-alphabet.zip',outdir='./fonts/')\n",
-    "patoolib.extract_archive('./fonts/acki-preschool.zip',outdir='./fonts/')\n",
-    "patoolib.extract_archive('./fonts/two-turtle-doves.zip',outdir='./fonts/')"
+    "You will probably want to play around with this sample to better understand a sample image's parameters before you later go on to tweak the sample generating function (which randomizes across a large space of possible parameters)."
    ]
```

Comment:
Adding the section above explains thy why behind the code and clarifies your processes.  Thank you for including.